cukinia

```diff
@@ -821,6 +822,18 @@ _cukinia_device()
 	test -d $device
 }
 
+# _cukinia_knoerror: Check the kernel has booted without important errors
+# arg1: the message priority
+_cukinia_knoerror()
+{
+	local priority="$1"
+	local ret
+	_cukinia_prepare "Checking that kernel has booted with no error (priority: $priority)"
+	journalctl --dmesg --priority $priority | grep "No entries"
```

Comment:
Considering all systems don't have systemd, we should check for it, use journalctl if available, and fallback on dmesg